src/patches/dense_patch_computation.cpp

```diff
@@ -413,6 +466,47 @@ InstructionApplicationResult try_apply_instruction_direct_followup(
             return {std::make_unique<std::runtime_error>(
                     lstk::cat(instruction,";Could not get magic state")), {}};
 
+    }
+    else if (auto* yr = std::get_if<YStateRequest>(&instruction.operation))
+    {
+        // Unbind Y state patch if already bound
+        if (slice.get_patch_by_id(yr->target))
+        {
+            slice.get_patch_by_id(yr->target)->get().id = std::nullopt;
+            return{nullptr, {}};
+        }
+        // Otherwise, get minimum (L1) distance unbound Y state patch
+        else 
+        {
+            std::optional<Cell> min_cell;
+            double min_dist = 9999; 
+            double dist;
```

Comment:
Can init where used

Reply:
Can you clarify what you mean?

Reply:
The variable is declared once scope outside of where it is used